src/main/resources/timeline/article/index.scala.html

```diff
@@ -1,20 +1,39 @@
 @import com.gu.contentatom.renderer.utils.GuardianDateFormatter
-@(atom: com.gu.contentatom.thrift.Atom
-, data: com.gu.contentatom.thrift.atom.timeline.TimelineAtom
+@import com.gu.contentatom.thrift.atom.timeline.TimelineItem
+@(atom: com.gu.contentatom.thrift.Atom,
+        data: com.gu.contentatom.thrift.atom.timeline.TimelineAtom
 )
 
+@renderFormattedDate(item: TimelineItem, format: String) = @{
```

Comment:
I'm thinking this could go in an implicit class around `TimelineItem` to avoid adding logic to the template